va/caa.go

```diff
@@ -40,12 +40,23 @@ func (va *ValidationAuthorityImpl) IsCAAValid(
 func (va *ValidationAuthorityImpl) checkCAA(
 	ctx context.Context,
 	identifier core.AcmeIdentifier) *probs.ProblemDetails {
-	present, valid, err := va.checkCAARecords(ctx, identifier)
+	present, valid, records, err := va.checkCAARecords(ctx, identifier)
 	if err != nil {
 		return probs.DNS("%v", err)
 	}
-	va.log.AuditInfof("Checked CAA records for %s, [Present: %t, Valid for issuance: %t]",
-		identifier.Value, present, valid)
+
+	// Build a string with the DNS presentation format of each *dns.CAA record we
+	// checked (if any) separated by commas to be included in the audit log line
+	var recordsStr strings.Builder
```

Comment:
Let's log a JSON object instead, which has better defined semantics for separate fields and escaping quotes.

Reply:
SGTM

Reply:
@jsha Addressed in 7556d5d